src/replaying.jl

```diff
@@ -42,6 +41,22 @@ end
 end
 
 
+const WHITE_LIST = [
+    Base.promote_op, Base.to_shape,
+    Core.getfield,
+    Core.:(===),
+    Base.iterate,
+    Broadcast.broadcasted, Broadcast.instantiate,
```

Comment:
won't blocking `Broadcast.instantiate` cause a ton of allocations to be missed?

Reply:
but the allocation happens in `materialize!` which is the `copyto!` function IIUC?